alembic/versions/3c66b72cc8c3_new_migration.py

```diff
@@ -0,0 +1,72 @@
+"""New migration
+
+Revision ID: 3c66b72cc8c3
+Revises: 945688b6e665
+Create Date: 2024-10-31 14:50:36.987865
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = '3c66b72cc8c3'
+down_revision: Union[str, None] = '945688b6e665'
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('authors',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('name', sa.String(length=255), nullable=True),
+    sa.Column('bio', sa.String(length=1023), nullable=True),
+    sa.PrimaryKeyConstraint('id'),
+    sa.UniqueConstraint('name')
+    )
+    op.create_index(op.f('ix_authors_id'), 'authors', ['id'], unique=False)
+    op.create_table('books',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('title', sa.String(length=255), nullable=False),
+    sa.Column('summary', sa.String(length=255), nullable=False),
```

Comment:
The `summary` column in the `books` table is set to `nullable=False`. Ensure this aligns with your data requirements, as it will require every book entry to have a summary.